### PR TITLE
Book detail 1 - Texts

### PR DESCRIPTION
**Pull request by Julianhm9612**
## Summary
- Completed the project setup
- Added a basic structure in HTML to the book detail
- Added some styles to format the texts
- Added a new file SCSS called vars and added new variables

## Screenshot
Captura de Pantalla 2021-05-10 a la(s) 5 13 10 p  m

## Trello Card
https://trello.com/c/zHx8REN5/26-detalle-de-libro-i-textos

### CODE REVIEW

File: books/index.html [hunk 1]
```diff
@@ -7,10 +7,32 @@
         <meta name="description" content="training">
         <meta name="author" content="Wolox">
         <link rel="stylesheet"  type="text/css" href="application.css">
-        <link rel="preconnect" href="https://fonts.gstatic.com">
-        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
     </head>
     <body>
-        <h1 class="title">Hello World</h1> 
+        <div class="book-detail">
+            <div class="book-box">
```
Comment (mcallegari10) [inline books/index.html]: I think this div may not be needed 🤔

File: books/index.html [hunk 2]
```diff
@@ -7,10 +7,32 @@
...
+                <img class="book-box-cover" src="" alt="Portada">
+                <div class="book-box-info">
+                    <div class="book-box-title">
+                        <h2>Título del libro</h2>
+                        <span>(género)</span>
+                    </div>
+                    <div class="details">
```
Comment (mcallegari10) [inline books/index.html]: This div is not needed

File: books/scss/detail.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
+.book-box {
+  .book-box-info {
+    .book-box-title {
+      h2 {
```
Comment (mcallegari10) [inline books/scss/detail.scss]: Remember to use only classes

**Review by mcallegari10 — CHANGES_REQUESTED** [review]
Remember to add an EOF line

File: books/scss/detail.scss (hunk 3 above)
Comment (Julianhm9612) [reply books/scss/detail.scss]: Classes have been added to improve this point

File: books/index.html (hunk 1 above)
Comment (Julianhm9612) [reply books/index.html]: Have been improved

File: books/index.html (hunk 2 above)
Comment (Julianhm9612) [reply books/index.html]: Have been improved

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**